src/app/view/components/_common/dot-push-publish-dialog/dot-push-publish-dialog.component.ts

```diff
@@ -111,6 +79,7 @@ export class DotPushPublishContentTypesDialogComponent implements OnInit, OnDest
      */
     close(): void {
         this.cancel.emit(true);
+        this.dialogShow = false;
```

Comment:
Shouldn't call the service here?

Reply:
the show hide of the dialog lives in the component.  the service the only send data.